tests/test_xrdfile.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of xrootdfs
+# Copyright (C) 2015 CERN.
+#
+# xrootdfs is free software; you can redistribute it and/or modify it under the
+# terms of the Revised BSD License; see LICENSE file for more details.
+
+"""Test of XRootDFS."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
```

Comment:
Can you fix the import sorting using isort (https://github.com/fisadev/vim-isort#installation). The ``.editorconfig`` already has the correct settings.